vagrant/provisioning/roles/arkcase-developer/tasks/templates/start_services.sh

```diff
@@ -3,7 +3,7 @@
 ##if the services are not running it will start the services
 
 ##list the services to check
-SERVICES=('haproxy' 'config-server' 'mariadb' 'httpd' 'pentaho' 'snowbound' 'solr' 'alfresco' 'activemq' 'confluent-kafka' 'confluent-zookeeper' 'confluent-control-center' 'confluent-kafka-connect' 'confluent-kafka-rest' 'confluent-ksql' 'confluent-kafka-connect' 'confluent-schema-registry' 'mongod' 'samba')
+SERVICES=('haproxy' 'config-server' 'mariadb' 'httpd' 'pentaho' 'snowbound' 'solr' 'alfresco' 'activemq' 'confluent-kafka' 'confluent-zookeeper' 'confluent-control-center' 'confluent-kafka-connect' 'confluent-kafka-rest' 'confluent-ksql' 'confluent-kafka-connect' 'confluent-schema-registry' 'mongod' 'samba' 'alfresco7')
```

Comment:
Should you remove alfresco?

Reply:
alfresco should be removed or alfresco7. It depends when we are going to start with alfresco7 as default installation

Reply:
This is for dev environments, so we should switch immediately. For production environments we should first test and then make it default.

Reply:
Pretty sure Alfresco 7 became default with DSO-480.  At least in arkcase-ce.yml and arkcase-dev.yml (pretty sure) the alfresco tags call the alfresco7 roles.